### ISSUE TITLE
[web] TextEditing error in a11y mode in Safari desktop

### ISSUE THREAD

**Issue by nturgut** [flutter/flutter#45199]
Text editing does not work. Tested on: Safari, MacOS, VoiceOver

possible reason: The key to enter the text editing area VO+UpArrow might not be recognized.

When the same test was conduct on Chrome + ChromeVox all worked correctly
